shortfin/examples/python/fastapi/server.py

```diff
@@ -56,11 +56,22 @@ async def run(self):
                     responder.send_response(
                         JSONResponse({"answer": request.request_value})
                     )
+                elif request.request_value == 2:
+                    count = 0
+                    while not responder.is_disconnected() and count < 5:
```

Comment:
Could we move this logic into `is_disconnected`? I.e. include a time disconnected before termination. We can default set to `1` but it feels reasonable to say `is_disconnected(timeout=1)`.